src/Polly.Core/Simmy/Behavior/BehaviorChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Polly.Simmy.Behavior;
```

Comment:
How about to expose all these extensions in just `Polly.Simmy`?

It would improve the discoverability of all Simmy extensions. This is what we did with core extensions, all of these are under the `Polly` namespace.